service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/ValidateEmergencyProtectionOrderControllerTest.java

```diff
@@ -0,0 +1,70 @@
+package uk.gov.hmcts.reform.fpl.controllers;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.test.autoconfigure.OverrideAutoConfiguration;
+import org.springframework.boot.test.autoconfigure.web.servlet.WebMvcTest;
+import org.springframework.test.context.ActiveProfiles;
+import uk.gov.hmcts.reform.ccd.client.model.AboutToStartOrSubmitCallbackResponse;
+import uk.gov.hmcts.reform.ccd.client.model.CaseDetails;
+import uk.gov.hmcts.reform.fpl.enums.EPOType;
+import uk.gov.hmcts.reform.fpl.model.Address;
+
+import java.time.LocalDateTime;
+import java.util.Map;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static uk.gov.hmcts.reform.fpl.enums.EPOType.PREVENT_REMOVAL;
+import static uk.gov.hmcts.reform.fpl.enums.EPOType.REMOVE_TO_ACCOMMODATION;
+
+@ActiveProfiles("integration-test")
+@WebMvcTest(UploadDocumentsController.class)
+@OverrideAutoConfiguration(enabled = true)
+class ValidateEmergencyProtectionOrderControllerTest extends AbstractControllerTest {
+    private static final LocalDateTime NOW = LocalDateTime.now();
```

Comment:
Not too fussed about it but what do you think about using the time configuration class? i.e. just having:
```
@Autowired
private Time time
```
Or something like that